PutJSONTogether/app.js

```diff
@@ -0,0 +1,70 @@
+var fs = require("fs");
+console.log("\n *START* \n");
+var content = "artists.json";
+
+var musicians = [];
+var newObj = {};
+
+function convertToInformation(content) {
+    fs.readFile(content, 'utf8', function readFileCallback(err, data){
+        if (err){
+            console.log(err);
+        } else {
+            obj = JSON.parse(data); //now it an object
+            arrayofObj = obj.table.length;
+
+            for(i = 0; i < obj.table.length; i++) {
+                newObj = {};
```

Comment:
Why don't you just initialize all the fields that we expect here with null instead of setting them below. Also a better name for the var would be great.